target/linux/generic/hack-5.4/230-openwrt_lzma_options.patch

```diff
@@ -27,11 +27,11 @@ Signed-off-by: Imre Kaloz <kaloz@openwrt.org>
  # ---------------------------------------------------------------------------
  
  quiet_cmd_lzma = LZMA    $@
--      cmd_lzma = { cat $(real-prereqs) | lzma -9; $(size_append); } > $@
+-      cmd_lzma = { cat $(real-prereqs) | $(LZMA) -9; $(size_append); } > $@
 +      cmd_lzma = { cat $(real-prereqs) | lzma e -d20 -lc1 -lp2 -pb2 -eos -si -so; $(size_append); } > $@
```

Comment:
Should we convert the "new" line to $(LZMA) here as well?
(This will require a manual merge for x86/011-tune_lzma_options.patch as well).

Reply:
@adschm - Would you mind making the proposed modification?

Reply:
I updated the PR based on your modifications to `target/linux/generic/hack-5.4/230-openwrt_lzma_options.patch` and `target/linux/x86/patches-5.4/011-tune_lzma_options.patch`

Reply:
New commit builds (x86_64) and runs (ipq806x).